packages/cache/jest.config.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+module.exports = {
```

Comment:
weird to have such a complicated config here; why not have a central config?

Reply:
I have fixed this!